jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1502,7 +1502,7 @@ public void setProperty(String key, String value, String source)
                 properties.setProperty("java.version.micro", Integer.toString(ver.getMicro()), "Deprecated");
 
                 // ALPN feature exists
-                properties.setProperty("runtime.feature.alpn", Boolean.toString(isMethodAvailable(javax.net.ssl.SSLParameters.class, "getApplicationProtocols", null)), source);
+                properties.setProperty("runtime.feature.alpn", Boolean.toString(ver.getPlatform() >= 9), source);
```

Comment:
This is wrong.
Java 8 has support too.
The previous version checked for the existence of a specific class to know if the ALPN support was backported.